weave

```diff
@@ -554,6 +554,11 @@ echo_addresses() {
     echo $1 $2 $3
 }
 
+peer_args() {
+  local joined=$(printf "&peer=%s" "${@}")
+  echo ${joined:1}
+}
```

Comment:
@dpw is there a better way to do this, ideally completely inside sh?

Also, this is the first use of a local var in the script. How do we feel about that?